### ISSUE TITLE
import/order plugin not works as expected?

### ISSUE THREAD

**Issue by olstenlarck** [benmosher/eslint-plugin-import#1019]
Consider the following example

```mjs
import fs from 'fs';
import path from 'path';
import sasa from 'isobject';

import xxxx from './foo.js';
```

and the config 

```
{
  'import/order': [
    'error',
    {
      groups: [
        'builtin',
        'external',
        'internal',
        'parent',
        'sibling',
        'index',
      ],
      'newlines-between': 'always-and-inside-groups',
    },
  ],
}
```

It does not reports that before `isobject` should have new line? When delete the newline after it, it correctly reports that there should have newlines between groups.

When it is set to `never` still not works as expected. Example

```mjs
import fs from 'fs';
import path from 'path';

import sasa from 'isobject';
import xxxx from './foo.js';
```

It not reports that it should not have empty line before it. But works when you set newline after it.

**Comment by olstenlarck** [comment]
Damn, my fault. It seems that it not works well in VSCode - when use the eslint cli it reports them correctly, but not shown correctly in the editor.

**Comment by olstenlarck** [comment]
Actually, realized one thing. New option for the `newlines-between` option that requires newlines between groups but not inside groups.

**Comment by ljharb** [comment]
A new issue for that would be great, thanks
